instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -20,18 +22,26 @@ const (
 	GRPCStatusCodeKey = attribute.Key("rpc.grpc.status_code")
 )
 
+// InterceptorFilter is a predicate used to determine whether a given request in
+// interceptor info should be traced. A InterceptorFilter must return true if
```

Comment:
Interceptor also emits metrics.

```suggestion
// interceptor info should be instrumented. A InterceptorFilter must return true if
```